app/MMB/src/list/CityImage.js

```diff
@@ -18,12 +18,13 @@ import ImageBadges from './ImageBadges';
 
 type Props = {|
   imageUrl: string,
-  bookingId: string,
+  bookingId: ?number,
   type: string,
   departureCity: string,
   arrivalCity: string,
-  date: Date,
+  date: ?Date,
```

Comment:
Just to make sure I understand - these changes are because of IDX nullable output, right?

Reply:
Yes, I could do some fallback with new Date(), but anyways, it is passed through moment() later, and moment(null/undefined) => moment(new Date())